src/features/configTable/configTableExample.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box } from "@chakra-ui/react";
+import ConfigTable, { Employee } from "./configTable";
+import { useGetConfigQuery } from "./configTableApi";
+
+const ConfigTableExampleView = () => {
+  const { data, isLoading: getConfigIsLoading } = useGetConfigQuery(null, {
```

Comment:
I do expect get and post to be at the same 'level'. in this case, I expect the get to be done inside the EmployeeConfigTable